homeassistant/components/sensor/thinkingcleaner.py

```diff
@@ -12,9 +12,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-REQUIREMENTS = ['https://github.com/TheRealLink/pythinkingcleaner'
```

Comment:
The diff between versions is just pypi setup: https://github.com/TheRealLink/pythinkingcleaner/commits/master